rskj-core/src/test/java/co/rsk/peg/BridgeSupportTestPowerMock.java

```diff
@@ -1446,6 +1447,7 @@ public void registerBtcTransactionMigrationTx() throws BlockStoreException, Addr
     }
 
     @Test
+    @Ignore
```

Comment:
Do we have a story to modify this test case that are ignored now? 
Are you included new test cases for the function that you are moved? 
Remember not use Power mock for that. Thank you

Reply:
No Story, But I'm Working On The New Test Cases
Thanks Pam